tfhe/src/typed_api/booleans/client_key.rs

```diff
@@ -0,0 +1,25 @@
+use crate::boolean::client_key::ClientKey;
+
+use serde::{Deserialize, Serialize};
+
+use super::{parameters::BooleanParameterSet, FheBoolParameters, types::r#static::StaticBoolParameters};
```

Comment:
could the "weirdness" with `r#static` be avoided by renaming the static module ?

Reply:
static_ would work fine for me

Reply:
or maybe static_types ? unsure

Reply:
that would be `typed_apu/{boolean, shortint, integer}/types/static_types.rs`